[Bug] Simple DSL example cannot be rendered with Helm

## Description
Following the example from README in https://github.com/ergon/airlock-helm-charts/tree/master/charts/microgateway#simple-dsl-configuration:
* There is a Helm chart render error in `microgateway/templates/configmap.yaml:91:29` when enabling `config.simple.mapping.deny_rules`, see below

## To Reproduce
1. Create a Helmfile:
```yaml
repositories:
  - name: ergon
    url: https://ergon.github.io/airlock-helm-charts/
  - name: ealenn
    url: https://ealenn.github.io/charts/
  - name: bitnami
    url: https://charts.bitnami.com/bitnami

releases:
  - name: microgateway-test
    namespace: ergo-25
    chart: ergon/microgateway
    version: 0.6.0
    values:
      - config:
          global:
            ip_header:
              trusted_proxies:
                - 10.0.0.0/28
          simple:
            mapping:
              entry_path: /
              operational_mode: integration
              deny_rules:
                - level: strict
                  enabled: true
                  exceptions:
                    - parameter_name:
                        pattern: ^content$
                        ignore_case: true
                      path:
                        pattern: ^/mail/
                      method:
                        pattern: ^POST$
```
2. Execute `helmfile template`

## Expected behavior
`microgateway/templates/configmap.yaml` can be rendered, so that the ConfigMap is producing a valid YAML to apply for Kubernetes

## Current behavior
The ConfigMap cannot be rendered due to a Gotemplate syntax error: `executing "microgateway/templates/configmap.yaml" at <$simple.mapping.deny_rules.enabled>: can't evaluate field enabled in type interface {}`

## Screenshots / Logs
```console
...
Templating release=microgateway-test, chart=/tmp/525002622/microgateway-test/0.6.0/ergon/microgateway/microgateway
in ./config-mwe.yaml: command "/usr/local/bin/helm" exited with non-zero status:

PATH:
  /usr/local/bin/helm

ARGS:
  0: helm (4 bytes)
  1: template (8 bytes)
  2: microgateway-test (22 bytes)
  3: /tmp/525002622/microgateway-test/0.6.0/ergon/microgateway/microgateway (75 bytes)
  4: --namespace (11 bytes)
  5: ergo-25 (7 bytes)
  6: --values (8 bytes)
  7: /tmp/values561474757 (20 bytes)

ERROR:
  exit status 1

EXIT STATUS
  1

STDERR:
  coalesce.go:196: warning: cannot overwrite table with non table for deny_rules (map[enabled:true exceptions:[] level:standard log_only:false])
  coalesce.go:196: warning: cannot overwrite table with non table for deny_rules (map[enabled:true exceptions:[] level:standard log_only:false])
  Error: template: microgateway/templates/deployment.yaml:17:28: executing "microgateway/templates/deployment.yaml" at <include (print .Template.BasePath "/configmap.yaml") .>: error calling include: template: microgateway/templates/configmap.yaml:91:29: executing "microgateway/templates/configmap.yaml" at <$simple.mapping.deny_rules.enabled>: can't evaluate field enabled in type interface {}
```

## Additional information
The faulty code is probably here somewhere: https://github.com/ergon/airlock-helm-charts/blob/master/charts/microgateway/templates/configmap.yaml#L91

## Versions
 - Microgateway: irrelevant
 - Helm chart: 0.6.0
 - Helm client: v3.1.2
 - Kubernetes / Openshift: irrelevant

## Checklist
- [x] The issue contains all necessary information a third party requires to understand.
- [x] The documentation has already been checked.

--- a/Makefile
+++ b/Makefile
@@ -1,0 +1,3 @@
+helm-docs:
+	@echo --- Generating Chart READMEs
+	@docker run --rm -v $$(pwd):/helm-docs -u $$(id -u) jnorwood/helm-docs:v0.13.0

--- a/charts/microgateway/README.md
+++ b/charts/microgateway/README.md
@@ -216,7 +216,8 @@ config:
       ---------------------
 ```
 
-2. Deploy the Microgateway with the license.yaml file:
+2. [Create the image pull secret](#credentials-to-pull-image-from-docker-registry) to pull the microgateway image.
+3. Deploy the Microgateway with the license.yaml file:
   ```console
   helm upgrade -i microgateway airlock/microgateway -f license.yaml
   ```
@@ -344,15 +345,15 @@ By default, the Airlock Microgateway is configured with the [Simple DSL configur
         entry_path: /
         operational_mode: integration
         deny_rules:
-          - level: strict
-            exceptions:
-              - parameter_name:
-                  pattern: ^content$
-                  ignore_case: true
-                path:
-                  pattern: ^/mail/
-                method:
-                  pattern: ^POST$
+          level: strict
+          exceptions:
+            - parameter_name:
+                pattern: ^content$
+                ignore_case: true
+              path:
+                pattern: ^/mail/
+              method:
+                pattern: ^POST$
       backend:
         protocol: https
         hostname: custom-backend-service
@@ -408,15 +409,15 @@ The use cases outlined above can also occur slightly differently. But all of the
               operational_mode: integration
               session_handling: enforce_session
               deny_rules:
-                - level: standard
-                  exceptions:
-                    - parameter_name:
-                        pattern: ^content$
-                        ignore_case: true
-                      path:
-                        pattern: ^/mail/
-                      method:
-                        pattern: ^POST$
+                level: standard
+                exceptions:
+                  - parameter_name:
+                      pattern: ^content$
+                      ignore_case: true
+                    path:
+                      pattern: ^/mail/
+                    method:
+                      pattern: ^POST$
             - name: api
               entry_path: /api/
               session_handling: ignore_session
@@ -624,7 +625,7 @@ In case that multiple hosts are configured, TLS-SNI is used to distinguish what 
   ```
 
 ### Openshift Route
-Since the Route is already available in an Openshift environment, nothing has to be installed additionally.
+Since the Route controller is already available in an Openshift environment, nothing has to be installed additionally.
 
 #### Route terminating HTTP
 

--- a/charts/microgateway/README.md.gotmpl
+++ b/charts/microgateway/README.md.gotmpl
@@ -123,7 +123,8 @@ config:
       ---------------------
 ```
 
-2. Deploy the Microgateway with the license.yaml file:
+2. [Create the image pull secret](#credentials-to-pull-image-from-docker-registry) to pull the microgateway image.
+3. Deploy the Microgateway with the license.yaml file:
   ```console
   helm upgrade -i microgateway airlock/microgateway -f license.yaml
   ```
@@ -248,15 +249,15 @@ By default, the Airlock Microgateway is configured with the [Simple DSL configur
         entry_path: /
         operational_mode: integration
         deny_rules:
-          - level: strict
-            exceptions:
-              - parameter_name:
-                  pattern: ^content$
-                  ignore_case: true
-                path:
-                  pattern: ^/mail/
-                method:
-                  pattern: ^POST$
+          level: strict
+          exceptions:
+            - parameter_name:
+                pattern: ^content$
+                ignore_case: true
+              path:
+                pattern: ^/mail/
+              method:
+                pattern: ^POST$
       backend:
         protocol: https
         hostname: custom-backend-service
@@ -312,15 +313,15 @@ The use cases outlined above can also occur slightly differently. But all of the
               operational_mode: integration
               session_handling: enforce_session
               deny_rules:
-                - level: standard
-                  exceptions:
-                    - parameter_name:
-                        pattern: ^content$
-                        ignore_case: true
-                      path:
-                        pattern: ^/mail/
-                      method:
-                        pattern: ^POST$
+                level: standard
+                exceptions:
+                  - parameter_name:
+                      pattern: ^content$
+                      ignore_case: true
+                    path:
+                      pattern: ^/mail/
+                    method:
+                      pattern: ^POST$
             - name: api
               entry_path: /api/
               session_handling: ignore_session
@@ -528,7 +529,7 @@ In case that multiple hosts are configured, TLS-SNI is used to distinguish what 
   ```
 
 ### Openshift Route
-Since the Route is already available in an Openshift environment, nothing has to be installed additionally.
+Since the Route controller is already available in an Openshift environment, nothing has to be installed additionally.
 
 #### Route terminating HTTP
 

